demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -665,6 +665,9 @@ def is_valid_fetch(self) -> bool:
         fetch_params_exist = True
         if self.current_file.get('script', {}).get('isfetch') is True:
             params = [_key for _key in self.current_file.get('configuration', [])]
+            for counter, param in enumerate(params):
+                if 'defaultvalue' in param:
+                    params[counter].pop('defaultvalue')
```

Comment:
this should work as the `param` is a reference and not a copy
```suggestion
            for param in params:
                if 'defaultvalue' in param:
                    param.pop('defaultvalue')
```